torch/_dynamo/convert_frame.py

```diff
@@ -215,13 +215,14 @@ def has_tensor(obj):
         if has_tensor(value):
             return True
 
-    log.debug(
-        "skipping because no torch.* %s \
-            %s %s",
-        frame.f_code.co_name,
-        frame.f_code.co_filename,
-        frame.f_code.co_firstlineno,
-    )
+    if config.verbose:
```

Comment:
maybe this one is fine, it is not as verbose as when we trace through TorchDynamoContext